provider/datagen/src/transform/icuexport/normalizer/mod.rs

```diff
@@ -91,13 +91,15 @@ macro_rules! normalization_tables_provider {
             DecompositionTables,
             $file_name,
             {
-                let mut scalars24: Vec<char> = Vec::new();
-                for &u in toml_data.scalars32.iter() {
-                    scalars24.push(
+                let scalars24 = toml_data
+                    .scalars32
+                    .iter()
+                    .map(|&u| {
```

Comment:
You can collect `Iterator<Result<T, E>>` into `Result<Vec<T>, E>` and then `?` that

Reply:
That's a neat trick 👍 Thanks, fixed.